File: 1. Two Sum.js
var twoSum = function(nums, target) {
    var answer = [0, 0];
    for (var i = 0; i < nums.length - 1; i++) {
        for (var j = i + 1; j < nums.length; j++) {
            if (nums[i] + nums[j] == target) {
                answer[0] = i;
                answer[1] = j;
            }
        }
    }
    return answer;
}

File: 1. Two Sum.py
def two_sum(nums, target):
    result = []
    for index in range(len(nums) - 1):
        for indexplus in range(index + 1, len(nums)):
            if nums[index] + nums[indexplus] == target:
                result.append(index)
                result.append(indexplus)
                return result
    

print(two_sum([1, 2, 3, 4, 5, 6, 7, 8, 9], 5))